analysis/contribution_bounders.py

```diff
@@ -61,11 +61,14 @@ def bound_contributions(self, col, params, backend, report_generator,
         def rekey_per_privacy_id_per_partition_key_and_unnest(pid_pk_v_values):
             privacy_id, partition_values = pid_pk_v_values
             num_partitions_contributed = len(partition_values)
+            num_contributions = sum(
+                (len(values) for _, values in partition_values))
             for partition_key, values in partition_values:
                 if sampler is not None and not sampler.keep(partition_key):
                     continue
                 yield (privacy_id, partition_key), (len(values), sum(values),
-                                                    num_partitions_contributed)
+                                                    num_partitions_contributed,
+                                                    num_contributions)
```

Comment:
Why is L0LinfBounder returning num_contributions? Shouldn't we have a separate bounder in this file for this?